README.md

```diff
@@ -417,6 +417,12 @@ Sets `docker-compose` to run with `--verbose`
 
 The default is `false`.
 
+### `all-logs` (optional)
+
+If set to false, only upload failed container logs.
+
+The default is `true`.
```

Comment:
You probably want to add the rm option in the docs as well

Reply:
Yep, it's on my TODO section. I wanted to check first if this was OK or not.